lib/loader.js

```diff
@@ -57,10 +57,11 @@ module.exports = function (content) {
 
   var bubleOptions = hasBuble && options.buble ? '?' + JSON.stringify(options.buble) : ''
   var defaultLoaders = {
-    html: templateCompilerPath + '?id=' + moduleId,
+    html: '',
     css: (isServer ? '' : styleLoaderPath + '!') + 'css-loader' + (needCssSourceMap ? '?sourceMap' : ''),
     js: hasBuble ? ('buble-loader' + bubleOptions) : hasBabel ? 'babel-loader' : ''
   }
+  var templateCompilerLoaderCommand = templateCompilerPath + '?id=' + moduleId
```

Comment:
Why assign this as a separate variable here? We can just use `defaultLoaders.html` in both places